src/ripple/app/paths/PathRequest.cpp

```diff
@@ -408,6 +446,110 @@ void PathRequest::resetLevel (int l)
         iLastLevel = l;
 }
 
+bool
+PathRequest::findPaths (
+    RippleLineCache::ref cache,
+        FindPaths& fp, Issue const& issue,
+            Json::Value& jvArray)
+{
+    // TODO MPORTILLA Is it necessary to cache this iusse's STPathSet?
+    // Is the unbound map growth a potential problem?
+    STPath fullLiquidityPath;
```

Comment:
mContext can only grow per issue with additional pathsets. Seems possible to submit many requests with many source currencies and accounts causing unbound consumption.